backtrace-library/src/main/java/backtraceio/library/services/BacktraceMetrics.java

```diff
@@ -272,7 +279,7 @@ public boolean addUniqueEvent(String attributeName, Map<String, Object> attribut
             return false;
         }
 
-        Map<String, String> metricsAttributes = ReportDataBuilder.getReportAttribues(attributes).first;
+        Map<String, String> metricsAttributes = ReportDataBuilder.getReportAttributes(attributes, true).getAttributes();
```

Comment:
it's a bit pointless to collect all attributes and annotations and discard the annotations. Maybe it's worth to create a separate method to get only attributes?

Reply:
in the metrics context, we won't have too many annotations. We can ofc have a separated method only for attributes, however right now it won't provide too many benefits.